src/improvements/template/boilerplate/OPCMBaseTask.template.sol

```diff
@@ -0,0 +1,103 @@
+        vm.label(OPCM, "OPCM");
+
+        require(false, "TODO: Perform a StandardValidatorV200 version check e.g. see comments below.");
+        // STANDARD_VALIDATOR_V200 = IStandardValidatorV200(tomlContent.readAddress(".addresses.StandardValidatorV200"));
+        // require(STANDARD_VALIDATOR_V200.disputeGameFactoryVersion().eq("1.0.1"), "Incorrect StandardValidatorV200");
+        // vm.label(address(STANDARD_VALIDATOR_V200), "StandardValidatorV200");
+
+        require(false, "TODO: Implement with the correct template setup.");
+    }
+
+    /// @notice A single call to OPCM.upgrade() is made for all l2 chains.
+    function _build() internal override {
+        require(false, "TODO: Implement with the correct build logic.");
+        SuperchainAddressRegistry.ChainInfo[] memory chains = superchainAddrRegistry.getChains();
+        IOPContractsManager.OpChainConfig[] memory opChainConfigs =
+            new IOPContractsManager.OpChainConfig[](chains.length);
+
+        for (uint256 i = 0; i < chains.length; i++) {
+            opChainConfigs[i] = IOPContractsManager.OpChainConfig({
+                systemConfigProxy: ISystemConfig(superchainAddrRegistry.getAddress("SystemConfigProxy", chains[i].chainId)),
+                proxyAdmin: IProxyAdmin(superchainAddrRegistry.getAddress("ProxyAdmin", chains[i].chainId)),
+                absolutePrestate: absolutePrestates[chains[i].chainId]
+            });
+        }
+
+        // See: template/OPCMUpgradeV200.sol for more information on why we expect a revert here.
+        (bool success,) = OPCM.call(abi.encodeCall(IOPContractsManager.upgrade, (opChainConfigs)));
+        require(!success, "OPCMUpgradeV200: Call unexpectedly succeeded; expected revert due to non-delegatecall.");
+    }
+
+    /// @notice This method performs all validations and assertions that verify the calls executed as expected.
+    function _validate(VmSafe.AccountAccess[] memory, Action[] memory) internal view override {
+        SuperchainAddressRegistry.ChainInfo[] memory chains = superchainAddrRegistry.getChains();
+        for (uint256 i = 0; i < chains.length; i++) {
+            require(false, "TODO: Implement with the correct validation logic.");
+            require(false, "TODO: Call StandardValidator.validate()");
+        }
+    }
+
+    /// @notice Override to return a list of addresses that should not be checked for code length.
+    function getCodeExceptions() internal view virtual override returns (address[] memory) {
+        require(
+            false, "TODO: Implement the logic to return a list of addresses that should not be checked for code length."
+        );
+        return new address[](0);
+    }
+}
+
+/// TODO: If you need any interfaces from the Optimism monorepo submodule. Define them here instead of importing them.
+/// Doing this avoids tight coupling to the monorepo submodule and allows you to update the monorepo submodule
+/// without having to update the template.
```

Comment:
I'd recommend moving this comment higher up for visibility, I think it will often get missed

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/837/commits/a74edefd55c64196294ba7b3620b7cbf30ba8c32